perma_web/frontend/components/App.vue

```diff
@@ -1,12 +1,25 @@
 <script setup>
 import CreateLink from './CreateLink.vue';
-import { onBeforeMount } from 'vue'
-import { getLinksRemainingStatus, getUserTypes } from '../lib/store'
+import { onBeforeMount, watchEffect } from 'vue'
+import { getLinksRemainingStatus, getUserTypes, getOrganizationFolders, getSponsoredFolders } from '../lib/store'
+import { globalStore } from '../stores/globalStore';
 
 onBeforeMount(() => {
+    globalStore.updateLinksRemaining(links_remaining)
     getLinksRemainingStatus(links_remaining, is_nonpaying)
-    getUserTypes(is_individual)
+    getUserTypes(is_individual, current_user)
 })
+
+watchEffect(() => {
+    if (globalStore.userTypes.includes('orgAffiliated')) {
+        getOrganizationFolders()
+    }
```

Comment:
In the original implementation, it appears we fetch organization folders for both org and sponsored users — but the fetch doesn't return data for sponsored users. So I if I'm not mistaken, this more conditional fetch might make more sense?